hirosystems/stacks-blockchain-api/templates/api-writer/statefulset.yaml

```
@@ -160,9 +160,8 @@ spec:
             - |
               if [ -z "$(ls -A "${DATA_DIR}" | grep -v "lost+found")" ]; then
                 echo "Downloading archive"
-                wget ${ARCHIVE_URL} -O ${DATA_DIR}/archive.tar.gz
-                tar zxvf ${DATA_DIR}/archive.tar.gz -C ${DATA_DIR}
-                rm -f ${DATA_DIR}/archive.tar.gz
+                wget ${ARCHIVE_URL} -O ${DATA_DIR}/archive.gz
+                gzip -d ${DATA_DIR}/archive.gz && mv ${DATA_DIR}/archive ${DATA_DIR}/stacks-node-events.tsv
```

What you wrote should work, but alternatively you could do this without using `mv` and I believe this would still work. It wasn't written like this previously because `tar` would extract the file from the archive which happened to already be named correctly, whereas `gzip` uses the same file name as the compressed file, just without the `.gz` at the end.
```suggestion
                wget ${ARCHIVE_URL} -O ${DATA_DIR}/stacks-node-events.tsv.gz
                gzip -d ${DATA_DIR}/stacks-node-events.tsv.gz
```

---

cool, this works